_pages/api/clients.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: api_docs_article
+title:  "Clients"
+---
+
+Algorithmia clients are available as libraries for you to use inside your project.
+In addition to browsing the API specification to see language-specific usage for each API endpoint,
+client-specific guides are located in the [Algorithmia Developer Center]({{site.baseurl}}/clients).
+
+This table provides links to various resources for each client:
+
+<div class="syn-table-container" markdown="1">
```

Comment:
Can we add `overflow-x: scroll` to the `.syn-table-container` element? Otherwise the content is clipped on mobile with no way to see the right side.

Reply:
Yep! Updated!